src/node.cc

```diff
@@ -686,6 +674,11 @@ static void PlatformInit(ProcessInitializationFlags::Flags flags) {
   }
 #endif  // __POSIX__
 #ifdef _WIN32
+  {
+    constexpr ULONG first = TRUE;
+    per_process::old_vectored_exception_handler =
+        AddVectoredExceptionHandler(first, TrapWebAssemblyOrContinue);
+  }
```

Comment:
IIUC, shouldn't this code be wrapped within  `NODE_USE_V8_WASM_TRAP_HANDLER`?

Reply:
resolved